services/masterdata/src/lib/core/adapters/translation/languages-result-list-model.adapter.ts

```diff
@@ -0,0 +1,21 @@
+import { IAdapter } from "../../interfaces";
+import { LanguageItem, LanguagesResultList } from "../../models";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export class LanguagesResultListModelAdapter implements IAdapter<any, LanguagesResultList> {
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	adapt(source: any): LanguagesResultList {
```

Comment:
Dont use any here since its an adapter. We really cannot convert anything to a LanguageResultList :)